packages/devtools_app/test/test_infra/scenes/standalone_ui/editor_service/editor_server.dart

```diff
@@ -13,38 +17,46 @@ import 'package:devtools_app/src/service/editor/api_classes.dart';
 /// the editor services. It is the opposite of [EditorClient] which is for
 /// consuming the services provided by the editor(server).
 abstract class EditorServer {
```

Comment:
I've renamed these so that "FakeEditor" and "EditorServer" (and similar names) are now "SimulatedEditor" and "SimulatedEditorBase" in 60707d74c.

These classes are split up more than they'll ultimately need to be to support having two implementations (and the idea that EditorServer might be useful outside of DevTools). We can merge/flatten some of these classes when we remove the postMessage versions that I think will probably help clarify things much more too (there's a TODO about this).